ironfish/src/blockchain/blockchain.test.ts

```diff
@@ -887,6 +851,10 @@ describe('Blockchain', () => {
         reason: VerificationResultReason.DOUBLE_SPEND,
       })
     }
+
+    // The chain should re-org back to the valid chain once it sees the next block
+    await expect(nodeA.chain).toAddBlock(blockA6)
+    expect(nodeA.chain.head.hash.equals(blockA6.header.hash)).toBe(true)
   })
 
   it('does not remove nullifiers from double spends during reorg', async () => {
```

Comment:
we should delete this entire test now since double spends are not allowed. It doesn't test anything anymore.

Reply:
Yea agreed, removed it